tests/acceptance/25_cf-execd/dies_in_time.cf

```diff
@@ -0,0 +1,155 @@
+# ENT-3147, zd#3157
+
+# Ensure that cf-execd dies within 3 seconds after being signalled.
```

Comment:
This new acceptance test takes around 2 minutes to complete and it is too complex to convert to a `timed` test. But I don't like it running with all our other fast tests.

What I'm thinking is to create a `slow` category (i.e. directory) of tests and put this in. This category of tests will be disabl-able (?!) from our `testall` script.

What do you think @kacf @nickanderson ?